packages/synapse-react-client/src/synapse-queries/auth/useTwoFactorEnrollment.ts

```diff
@@ -43,9 +43,23 @@ export function useFinishTwoFactorEnrollment(
       if (options?.onSuccess) {
         options.onSuccess(...args)
       }
-      await queryClient.invalidateQueries({
-        queryKey: keyFactory.getTwoFactorAuthStatusQueryKey(),
-      })
+      await Promise.all([
```

Comment:
Cache invalidation (network) is one of the more complicated issues that we have to continue to contend with moving forward